layouts/partials/head.html

```diff
@@ -6,6 +6,7 @@
 {{- else -}}
 <meta name="ROBOTS" content="NOINDEX, NOFOLLOW">
 {{- end -}}
+{{- if eq hugo.Environment "production" -}}
```

Comment:
We also want this script for `nonprod` environments, I think.

Reply:
sure. did you edit the above comment by the way? I could have sworn you didn't mention `nonprod` before